IdentityCore/src/requests/sdk/msal/MSIDDefaultBrokerResponseHandler.m

```diff
@@ -83,6 +87,16 @@ - (MSIDBrokerResponse *)brokerResponseFromEncryptedQueryParams:(NSDictionary *)e
         return nil;
     }
     
+    // In case MSAL requests AT POP but Broker response Bearer, we need to update authScheme
```

Comment:
What would be the developer experience in case of this one? 
If service requires POP and MSAL returns bearer, this might create a loop between app and the service.